language/grounding/intro-grounding.ipynb

```diff
@@ -149,7 +149,7 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    chat_model = ChatModel.from_pretrained("chat-bison")
</pre><p>Should these model names be moved into variables with colab parameters to make it easier to use other models in the future?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>